### PR TITLE
added player files, environment files, Controller, Service, Repo

### PR DESCRIPTION
**Pull request by T-Kutzner**

### CODE REVIEW

File: backend/src/main/java/de/tk/mastermind/repositories/PlayerRepository.java
Comment (gossie) [inline backend/src/main/java/de/tk/mastermind/repositories/PlayerRepository.java:11]: You don't need to define that method. It is already present in the `MongoRepository`.

**Review by gossie — CHANGES_REQUESTED** [review]
The issues with the endless recursion and the `@DeleteMapping` need to be fixed before approval, because they would cause errors.
The other issues are more or less optional.

**Comment by T-Kutzner** [comment]
Please check fixed issues

**Review by gossie — APPROVED** [review]

**Pull request merged**
